freqtrade/plot/plotting.py

```diff
@@ -386,8 +386,9 @@ def generate_candlestick_graph(pair: str, data: pd.DataFrame, trades: pd.DataFra
     )
     fig.add_trace(candles, 1, 1)
 
-    if 'buy' in data.columns:
-        df_buy = data[data['buy'] == 1]
+    # TODO-lev: Needs short equivalent
```

Comment:
Already done in `lev-freqtradebot`

Reply:
@xmatthias Actually I don't think this is true, sorry that I wrote this

Reply:
I didn't intend to fix these here (none of the ones you commented on), nor remove the comments.
i think it's still good to keep these notes "there" until we have the "fix" or solution in place (basically the PR implementing that will remove these lines) - so we can make sure we're not forgetting these things.

There's a thought behind every "todo" (both from my side, and/or from other people's sides) - reminding to not forget stuff - so it's important to keep them or conciously decide "i did this - can remove it".